scss/detail.scss

```diff
@@ -1,74 +1,160 @@
-.container-book {
-  background-color: $white-secondary;
-  box-shadow: 4px 4px 10px 0 $shadow;
-  display: flex;
-  height: 416px;
-  width: 900px;
-}
+// variables
+$badge-height: 119px;
+$badge-width: 95px;
+$container-height: 416px;
+$container-width: 900px;
 
-.container-img {
-  position: relative;
-  height: 100%;
-  max-width: 309px;
-  padding: 24px;
-  width: 100%;
-
-  .badge {
-    display: flex;
-    height: 119px;
-    width: 95px;
-    position: absolute;
-    transform: rotate(15deg);
-    right: -6px;
-    top: 0;
-    padding: 11px;
-  }
+.back-arrow {
+  display: flex; 
+  width: 90%;
+  height: 20px;
+  margin-bottom: 33px;
+  cursor: pointer;
+  
 }
 
-.container-detail {
-  height: auto;
-  padding: 24px;
-  width: auto;
+.back-arrow::before {
+  content: url(./assets/back-arrow-20.png);
+  height: 20px;
+  width: 20px;
+  margin-right: 12px;
 }
 
-.header-detail {
-  align-items: center;
-  border-bottom: 4px solid $green-lemon;
+.container-book {
+  background-color: $white-secondary;
+  box-shadow: 4px 4px 10px 0 $shadow;
   display: flex;
-  padding-bottom: 12px;
-  width: 100%;
-}
+  height: $container-height;
+  width: $container-width;
 
-.title-book {
-  color: $black-default;
-  font-weight: 700;
-  margin-left: 8px;
-  font-size: 32px;
-  line-height: 32px;
-}
+  .container-img {
+    position: relative;
+    height: 416px;
+    max-width: 309px;
+    padding: 24px;
+    width: 309px;
+  
+    .badge {
+      display: flex;
+      height: $badge-height;
+      width: $badge-width;
+      position: absolute;
+      transform: rotate(15deg);
+      padding: 10px;
+      right: -5px;
+      top: 0px;
+    }
+  }
 
-.title-gender {
-  color: $grey;
-  font-weight: 700;
-  padding-left: 8px;
-  line-height: 28px;
-  font-size: 28px;
+  .container-detail {
+    padding: 24px;
+  
+    .header-detail {
+      align-items: center;
+      border-bottom: 4px solid $green-lemon;
+      display: flex;
+      padding-bottom: 12px;
+      width: 100%;
+    
+      .title-book {
+        color: $black-default;
+        font-weight: 700;
+        margin-left: 8px;
+        font-size: 32px;
+        line-height: 32px;
+      }
+      
+      .title-gender {
+        color: $grey;
+        font-weight: 700;
+        padding-left: 8px;
+        line-height: 28px;
+        font-size: 28px;
+      }
+    }
+  
+    .detail-form {
+      margin: 36px 0 0 8px;
+    
+      .detail-item {
+        color: $black-default;
+        font-weight: 700;
+        font-size: 20px;
+        line-height: 20px;
+      }
+    
+      .detail-description {
+        color: $grey;
+        margin-left: 8px;
+        font-size: 20px;
+        line-height: 20px;
+      }
+    }
+  }
 }
 
-.detail-form {
-  margin: 36px 0 0 8px;
-}
+@media(max-width: $breakpoint) {
+  .back-arrow {
+    width: calc(#{$container-width} - 564px);
+    height: 20px;
+    margin-top: 33px;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
```

Comment:
This rule is already defined at the base `.back-arrow` class, so you don't have to specify it again :wink: